src/core/services/interimElement/interimElement.js

```diff
@@ -696,8 +699,6 @@ function InterimElementProvider() {
         function showElement(element, options, controller) {
           // Trigger onShowing callback before the `show()` starts
           var notifyShowing = options.onShowing || angular.noop;
-          // Trigger onComplete callback when the `show()` finishes
-          var notifyComplete = options.onComplete || angular.noop;
```

Comment:
This is here because it is meant to be used for animations. It is run as soon as the `$animate.enter` Promise resolves. This enables the ability to trigger an animation that comes right after the transitionIn animation is complete.